extensions/functions_datetime.yaml

```
@@ -3,39 +3,117 @@
 scalar_functions:
   -
     name: extract
-    description: Extract portion of a date/time value.
+    description:  >-
+      Extract portion of a date/time value.
+      * YEAR Return the year of the timestamp.
+      * ISO_YEAR Return the ISO 8601 week-numbering year. First week of an ISO year has the majority (4 or more) of its
+        days in January.
+      * US_YEAR Return US epidemiological year. First week of US epidemiological year has the majority (4 or more) of
+        it's days in January. Last week of US epidemiological year has the year's last Wednesday in it.
+        US epidemiological week starts on Sunday.
+      * QUARTER Return the quarter from a date argument (1-4).
+      * MONTH Return the month from the date passed (1-12).
+      * DAY Return the day of the month (1-31).
+      * US_DAY_OF_WEEK Return the day of week where Sunday (1) to Saturday (7).
+      * ISO_DAY_OF_WEEK Return the day of week where Monday (1) to Sunday (7).
+      * DAY_OF_YEAR Return the day of year (1 to 366). January 1st maps to day number 1, February 1st to 32, etc.
+      * WEEK Return week of year number (1-53). First week starts on first Monday of January.
+      * US_WEEK Return US the week number (1-53). First US week starts on first Sunday of January.
+      * ISO_WEEK Return ISO week of year number (1-53). First ISO calendar week has the majority (4 or more) of it's
+        days in January. ISO week starts on Monday.
+      * HOUR Return the hour (0-23).
+      * MINUTE Return the minute (0-59).
+      * SECOND Return the second (0-59).
+      * MILLISECOND Return number of milliseconds since the last full second.
+      * MICROSECOND Return number of microseconds since the last full millisecond.
+      * SUBSECOND Return number of microseconds until the next full second of the given timestamp.
```

That means it counts *down*...?

---

O sorry, that's wrong. How about:
```suggestion
      * SUBSECOND Return number of microseconds since the last full second of the given timestamp.
```

---

That looks correct (but still needs to be committed).